app/modules/transcribe/subjects.factory.js

```diff
@@ -39,7 +39,8 @@ function SubjectsFactory($q, AnnotationsFactory, localStorageService, zooAPI, zo
             _updateStorage();
         }
         if (!_queue.length) {
-            return _populateQueue()
+            return  _getSubjectSetId()
```

Comment:
is this a new function? i can't find the def anywhere? I assume this is linked to the `getRandomWorkflowAssociatedSubjectSets` function.

Reply:
Sorry my bad. It's there now.